Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/Scripts/Exporters/TemplatedExporter/TemplatedTopLevelDependenciesProjectExporter.cs

```diff
@@ -48,15 +51,15 @@ internal class TemplatedTopLevelDependenciesProjectExporter : ITopLevelDependenc
 
         public void Write()
         {
-            TemplatedWriter propsWriter = new TemplatedWriter(propsTemplateFile);
-
-            propsWriter.Write(ProjectGuidToken, Guid);
+            TemplatedWriter propsWriter = new TemplatedWriter(propsTemplateFile)
+                .Write(ProjectGuidToken, Guid)
+                .Write(MSBuildForUnityVersionToken, MSBuildForUnityVersion.ToString(3));
```

Comment:
Nit: Is there any way to share the MSB4U_VERSION and fieldCount so that we don't get inconsistencies between the two files? Otherwise someone could update one of these to 4 and forget to update the others, and I'd imagine debugging that could be fun.

Reply:
Changed ToString, but the MSB4U_VERSION is specific to the file template, so leaving it.